### PR TITLE
Add NV12 with SCANOUT flag

### PR DESCRIPTION
**Pull request by js0701**
DRM_FORMAT_NV12 is supported by overlay since Gen9

### CODE REVIEW

File: i915.c [hunk 1]
```diff
@@ -178,6 +181,8 @@ static int i915_add_combinations(struct driver *drv)
 	if (ret)
 		return ret;
 
+	drv_modify_combination(drv, HAL_PIXEL_FORMAT_NV12, &metadata, BO_USE_SCANOUT | BO_USE_RENDERING | BO_USE_TEXTURE);
```
Comment (kalyankondapally) [inline i915.c]: why do we need this ?

Comment (js0701) [inline i915.c]: @kalyankondapally  This is I thought what we talked about to make sure this format is Y-tiled?

Comment (kalyankondapally) [inline i915.c]: @js0701 I think we can remove this for now.

**Comment by kalyankondapally** [comment]
merged

**Pull request closed without merging**